packages/api-elements/lib/elements/Transition.js

```diff
@@ -116,6 +129,16 @@ class Transition extends ArrayElement {
   get transactions() {
     return this.children.filter(item => item.element === 'httpTransaction');
   }
+
+  /**
+   * @name host
+   * @type StringElement
+   * @memberof ParseResult.prototype
+   */
+  get host() {
+    return this.children
+      .filter(item => item.classes.contains('host'));
+  }
```

Comment:
```suggestion
```

I don't think we need this accessor, according to the design there won't be any hosts inside the content, only inside the `attributes` (handled above).

Reply:
ok, changed